lib/agent/providers/geo/index.js

```diff
@@ -1,44 +1,35 @@
-"use strict";
+const strategies = require('./strategies');
```

Comment:
let's take the opportunity to document this file

Reply:
This was documented in this PR https://github.com/prey/prey-node-client/pull/806